engine/storage/snapshot/src/main/java/org/apache/cloudstack/storage/snapshot/StorageSystemSnapshotStrategy.java

```diff
@@ -474,7 +473,7 @@ else if (volumeInfo.getFormat() == ImageFormat.OVA || volumeInfo.getFormat() ==
             try {
                 vmSnapshot = takeHypervisorSnapshot(volumeInfo);
             }
-            catch (ResourceAllocationException ex) {
+            catch (Exception ex) {
```

Comment:
@mike-tutkowski, Thanks for the fix. I think it would be better to capture the required exceptions with multiple catches instead of the parent Exception class. We can always put this at the end.